source/agora/consensus/EnrollmentManager.d

```diff
@@ -664,6 +667,68 @@ public class EnrollmentManager
         return false;
     }
 
+    /***************************************************************************
+
+        Get the key for the enrollment for this node
+
+        Returns:
+            the key for the enrollment
```

Comment:
Note that this function can "fail silently". No exception being thrown, and only a log message.
Originally I thought it was a bug, but it makes sense given the current design of the `EnrollmentManager`.
If that's intentional, it should be documented, and also the `log.error` should be downgraded to a non-error.